dbt/include/bigquery/macros/materializations/clone.sql

```diff
@@ -0,0 +1,9 @@
+{% macro bigquery__can_clone_table() %}
+    {{ return(True) }}
+{% endmacro %}
+
+{% macro bigquery__create_or_replace_clone(this_relation, defer_relation) %}
```

Comment:
Not a blocking comment, but why did we go with "defer"_relation? I get "this"_relation, because it likely aligns with `this` in the jinja context. But I don't get the reference of `defer`.

Reply:
I believe originally this was called `state_relation` but we didn't really have a hard set name I think `defer_relation` is just in reference to how this is a `defer state` type action is that correct @aranke?